hdr_html_canvas_element.md

```diff
@@ -125,13 +125,33 @@ The component signals are mapped to red, green and blue tristimulus values accor
 * Transfer function:
 
 ```
-   E = | E' / 12.92, if abs(E') ≤ 0.04045
-       | ((E' + 0.055) / 1.055)^2.4, otherwise
-
-       with E' ∈ ℝ
+decoding equation (non-linear to linear):
+
+if N < -0.04045
+  L = -1.0 * (((-1.0*N) + 0.055) / 1.055)^2.4;
+elseif (N >= -0.04045 && N <= 0.04045)
+  L  = N / 12.92;
+else
+  L = ((N + 0.055) / 1.055)^2.4;
+end
+
+encoding equation (linear to non-linear):
+
+if L < -0.0031308
+  N = -1.055 * (-1.0 * L) ^ (1.0/2.4) + 0.055;
+elseif (L >= -0.0031308 && L <= 0.0031308)
+  N  = L * 12.92;
+else
+  N = 1.055 * L ^ (1.0/2.4) - 0.055;
+end
 ```
 
-where `E'` is the non-linear colour value and `E` is the linear colour value
+where `N` is the non-linear colour value and `L` is the linear colour value
+
+![extended-srgb-decoding-function](https://user-images.githubusercontent.com/37905569/121105535-25ae7700-c7b9-11eb-92fd-a90e68cac7f3.png)
```

Comment:
(same remarks about plots as in [PR 52](https://github.com/w3c/ColorWeb-CG/pull/52))

Reply:
2021-07-19: either have graphs for all functions or none.